projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -162,6 +163,10 @@ public function post_page_metabox_assets() {
 			return;
 		}
 
+		$is_atomic_site = ( new Host() )->is_woa_site();
+		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
```

Comment:
We can use the `Host` class for this as well.

```suggestion
		$is_simple_site = ( new Host() )->is_wpcom_simple();
```